Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -1,6 +1,6 @@
 prototype(Neos.NeosIo:Reference.List) < prototype(TYPO3.Neos:Content) {
 	references = TYPO3.TypoScript:Collection {
-		collection = ${Search.query(site).nodeType('Neos.NeosIo:Reference').sortDesc('datePublished').execute()}
+		collection = ${Search.query(site).nodeType('Neos.NeosIo:Reference').sortDesc('datePublished').limit(500).execute()}
```

Comment:
Without the limit(500) only the first 8 references were shown.
In the long term we should add pagination and custom filtering